runtime/metrics.go

```diff
@@ -191,16 +191,8 @@ func (m *Metrics) recordEvent(evt *Event) {
 		tags[k] = v
 	}
 
-	if evt.Outcome != "" {
-		tags["outcome"] = string(evt.Outcome)
-	}
-
-	fields := map[string]interface{}{
-		"error": evt.Error,
-	}
```

Comment:
Are we OK not sending the error as a field? I believe dropping the outcome is OK because it's now part of the event type.

Reply:
I don't think we ever used `error` as a field - doesn't make sense to send it to a time-series DB in my opinion.